Калькулятор/calc.js

```diff
@@ -0,0 +1,71 @@
+let a = '';
```

Comment:
what are those random names `a` and `b`? please use descriptive names for variables

Reply:
examples here https://teamtreehouse.com/library/javascript-basics/naming-variables